src/remote/remote.go

```diff
@@ -57,12 +58,13 @@ var remoteCacheReadDuration = metrics.NewHistogram(
 //
 // It provides a higher-level interface over the specific RPCs available.
 type Client struct {
-	client      *client.Client
-	fetchClient fpb.FetchClient
-	initOnce    sync.Once
-	state       *core.BuildState
-	err         error // for initialisation
-	instance    string
+	client         *client.Client
+	remoteFSClient remotefs.Client
```

Comment:
Might've been nice to make these guys the same but i think the Client god object from their SDK is a bit too big to wrap up like this.